### PR TITLE
Change STOPSIGNAL: SIGINT

### PR DESCRIPTION
**Pull request by tranhuucuong91**
- Change `STOPSIGNAL`: `SIGINT`
- Update `docker-entrypoint.sh`: allow `docker run` custom command.
Ex: `docker run phpmyadmin/phpmyadmin sh`

### CODE REVIEW

**Comment by tranhuucuong91** [comment]
I want to change `STOPSIGNAL` from `SIGTERM` (default) to `SIGINT`.
The container will not have to wait for `TIMEOUT` to turn off.

P/S: Sorry for spam pull requests. I can not delete it.

File: Dockerfile [hunk 1]
```diff
@@ -7,12 +7,14 @@ RUN curl --location https://www.phpmyadmin.net/downloads/phpMyAdmin-latest-all-l
  && rm -rf /www/js/jquery/src/ /www/examples /www/po/
 
 COPY config.inc.php /www/
-COPY run.sh /run.sh
-RUN chmod u+rwx /run.sh
+COPY docker-entrypoint.sh /docker-entrypoint.sh
```
Comment (nijel) [inline Dockerfile]: chmod is not needed anymore? It seems that at least in some setups it is, see eg. https://github.com/klaemo/docker-couchdb/issues/19

**Comment by nijel** [comment]
Also on Travis this fails with "Unknown instruction: STOPSIGNAL"

**Comment by nijel** [comment]
This should not be needed since #48.

**Pull request closed without merging**